### DISCUSSION TITLE
What am I missing ?

### DISCUSSION THREAD

**Discussion by piperpipuk** [Budibase/budibase#3553] [Help]
I have an MSSQL query, with a parameter

`select clientName from clients where id = {{ clientID }}`

![image](https://user-images.githubusercontent.com/11796003/143579073-50779d23-b490-47a6-850e-d47ff59eaae3.png)

My schema on my form is setting the clientID from the {{ URL.id }} on my view page, eg /client/:id

![image](https://user-images.githubusercontent.com/11796003/143579276-16d336dc-c3f2-4341-83b1-511defd134c0.png)

When I populate the field group or insert a manual text field, for example, my only available field is "clientID"

![image](https://user-images.githubusercontent.com/11796003/143579309-b8caff45-a88a-4f33-8dc9-f8957b140ace.png)

**Comment by mjashanks** [comment]
Hi @UKXL You are on a very old version of Budibase.

Are you using Digital Ocean? You should really update before I can help you with anything :)

**Reply by piperpipuk** [reply]
Hi, @mjashanks I have moved the app over to the hosted version, I'm seeing the same issue.

![image](https://user-images.githubusercontent.com/11796003/143763589-47598a50-7d21-4f00-8b4d-50eb112025e2.png)

**Reply by mjashanks** [reply]
Yes, I see what you mean - this looks like a bug. 

There is a workaround though. You can manually create each field on your form. On each field, set the `Field` setting to the Column name (case sensitive).

That will load the values into the input.

FYI, we are about to upgrade the MS SQL connector, so that it is as good as the MySQL and PGSQL ones. I.e. you will not have to write `UPDATE`, `INSERT`, `DELETE` and `SELECT one` statements. This will be available in the next few days.

P.S. thanks for moving to cloud and reporting back :)
